unicode_utils.gemspec

```diff
@@ -19,6 +19,7 @@ Gem::Specification.new do |g|
   g.required_ruby_version = ">= 1.9.1"
   g.author = "Stefan Lang"
   g.email = "langstefan@gmx.at"
+  g.license = "BSD-2"
```

Comment:
I believe this should be `BSD-2-Clause` as per https://spdx.org/licenses/BSD-2-Clause.html

Reply:
I agree.

Reply:
Changed in 65d7149e23dd50d2488181d058d36ada2992bccc